app-backup/dar/dar-2.6.13-r1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic
+
+DESCRIPTION="A full featured backup tool, aimed for disks"
+HOMEPAGE="http://dar.linux.free.fr/"
+SRC_URI="mirror://sourceforge/dar/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux"
+IUSE="curl dar32 dar64 doc gcrypt gpg lzo nls rsync threads xattr"
+
+RESTRICT="test" # need to be run as root
+
+RDEPEND="
+	>=sys-libs/zlib-1.2.3:=
+	app-arch/bzip2:=
+	app-arch/xz-utils:=
+	sys-libs/libcap
+	curl? ( net-misc/curl )
+	gcrypt? ( dev-libs/libgcrypt:0= )
+	gpg? ( app-crypt/gpgme )
+	lzo? ( dev-libs/lzo:= )
+	rsync? ( net-libs/librsync:= )
+	threads? ( dev-libs/libthreadar:= )
+	xattr? ( sys-apps/attr:= )
+"
+
+DEPEND="
+	${RDEPEND}
+"
+BDEPEND="
+	nls? (
+		sys-devel/gettext
+		virtual/libintl
+	)
+	doc? ( app-doc/doxygen )
+"
+
+REQUIRED_USE="?? ( dar32 dar64 )
+		gpg? ( gcrypt )"
+
+src_configure() {
+	# configure.ac is totally funked up regarding the AC_ARG_ENABLE
+	# logic.
+	# For example "--enable-dar-static" causes configure to DISABLE
+	# static builds of dar.
+	# Do _not_ use $(use_enable) until you have verified that the
+	# logic has been fixed by upstream.
+	local myconf=(
+		--disable-python-binding
+		--disable-upx
+		$(usex curl '' --disable-libcurl-linking)
+		$(usex dar32 --enable-mode=32 '')
+		$(usex dar64 --enable-mode=64 '')
+		$(usex doc '' --disable-build-html)
+		#$(usex examples --enable-examples '')
+		$(usex gcrypt '' --disable-libgcrypt-linking)
+		$(usex gpg '' --disable-gpgme-linking)
+		$(usex lzo '' --disable-liblzo2-linking)
+		$(usex nls '' --disable-nls)
+		$(usex rsync '' --disable-librsync-linking)
+		$(usex threads '' --disable-threadar)
+		$(usex xattr '' --disable-ea-support)
+	)
+
+	# Bug 103741
+	filter-flags -fomit-frame-pointer
+
+	myconf+=( --disable-dar-static --disable-static )
+
+	econf "${myconf[@]}"
+}
+
+src_install() {
+	emake DESTDIR="${D}" pkgdatadir="${EPREFIX}"/usr/share/doc/${PF}/html install
+
+	local DOCS=( AUTHORS ChangeLog NEWS README THANKS TODO )
+	einstalldocs
+
+	find "${ED}" -name '*.la' -delete || die
+	find "${ED}" -name '*.a' -delete || die
+
+	# Bug 729150
+	(cd "${ED}/usr/share/doc/${PF}/html/samples" && mkdir MyBackup && tar zxf MyBackup.sh.tar.gz -C MyBackup && rm MyBackup.sh.tar.gz) || die
```

Comment:
What's up with this monster? How can you tell which part fails now? Actually can you walk me through what's happening here, because there's most likely a better way to deal with it. 

If you look at *Makefile.in* file and grep for *gzip* you most likely start to see where this is coming from. We'd prefer a build-system patch. I believe simply emptying out `DIST_TARGETS` will fix it, but please do test with 
https://wiki.gentoo.org/wiki/Package_testing#make.conf

Reply:
It doesn't come from there.  There is a tar.gz file included in the distribution, please check `dar-2.6.13/doc/samples/MyBackup.sh.tar.gz`, it is not built by the build system, only copied as is.  `DIST_TARGETS` applies to `make dist`, not to `make install`.

I unpacked it because the bug is an automated check.  Other ways to fix it would be:
- patch the build system to unpack it at `make install` time
- or simply remove it from installation

How do you suggest I fix this bug?

Reply:
Hmm I see. Can you try calling `docompress -x` on it in `src_install`, which ignores the package-manager compression for it?

Is the package any relevant for the user, does it make sense installing it?

Reply:
After a bit of searching it seems unpacking it is not a solution, because that directory contains this page: http://dar.linux.free.fr/doc/samples/index.html

Unpacking `MyBackup.sh.tar.gz` would break the link in `index.html` anyway.  So if I remove it and someone really needs it, they can download it.  Is this acceptable?

Reply:
Also I'll check if this directory is related to that "examples" USE flag from earlier.

Reply:
Yes please, sounds good.